### PR TITLE
Fix gateway requests daily

### PR DESCRIPTION
**Pull request by guillaumemichel**
This is only a fix for the first gateway plot. The others will follow.

### CODE REVIEW

**Comment by guillaumemichel** [comment]
`ashby` has to be run again the day we merge this to resolve conflicts

File: config/plotdefs/gateway-requests-daily.yaml
Comment (lidel) [inline config/plotdefs/gateway-requests-daily.yaml:1]: @guillaumemichel would it be possible to include https://github.com/probe-lab/website/issues/107 as part of this, or would that be a follow-up PR?

Comment (guillaumemichel) [reply config/plotdefs/gateway-requests-daily.yaml:1]: Yes, I'll add it here 👍🏻

**Comment by lidel** [comment]
@guillaumemichel where are we with fixing https://probelab.io/ipfs/gateways/?  
Anything I can do to unblock?

**Comment by guillaumemichel** [comment]
The [number of daily requests plot](https://probelab.io/ipfs/kpi/#http-gateway-usage) is back on the IPFS KPIs page, but the [Gateways page](https://probelab.io/ipfs/gateways) is still hidden from the menu because there are no other accurate plots to show.

**Pull request merged**